app/public/js/gscan.js

```diff
@@ -57,6 +57,16 @@
             $('#theme-submit').prop('disabled', !$(this)[0].files.length);
         });
 
+        // Mobile Menu Trigger
+        $('.gh-nav-burger').click(function () {
```

Comment:
Added the hamburger code snippet, as it wasn't working. Not that I expect anyone to test themes on a mobile, but you never know 🤷🏼‍♀️